site/content/3.12/develop/integrations/spring-boot-arangodb.md

```diff
@@ -0,0 +1,67 @@
+---
+title: Spring Boot Starter ArangoDB
+menuTitle: Spring Boot Starter
+weight: 7
+description: ''
+archetype: default
+---
+
+The Spring Boot Starter for ArangoDB is a set of convenient dependency descriptors
+that you can include in your application.
```

Comment:
I like the description - let's move it to the front matter (without the period) and add the description shortcode

Reply:
Done.